cli/src/cmd/forge/sig_collision.rs

```diff
@@ -0,0 +1,100 @@
+use crate::{
+    cmd::{forge::build::CoreBuildArgs, Cmd},
+};
+use clap::Parser;
+use ethers::{
+    prelude::{
+        info::ContractInfo
+    },
+    solc::{
+        utils::canonicalize
+    }
+};
+use foundry_common::compile;
+use tracing::trace;
+
+#[derive(Debug, Clone, Parser)]
+pub struct SigCollisionArgs {
+    #[clap(
+        help = "The first of the two contracts for which to look method signature collisions in the form `(<path>:)?<contractname>`.",
+        value_name = "FIRST_CONTRACT"
+    )]
+    pub first_contract: ContractInfo,
+    
+    #[clap(
+        help = "The second of the two contracts for which to look method signature collisions in the form `(<path>:)?<contractname>`.",
+        value_name = "SECOND_CONTRACT"
+    )]
+    pub second_contract: ContractInfo,
```

Comment:
Could you please adhere to the style of the other Clap Parser structs? This would mean the help is turned into a doc comment, and `value_name` is removed as it's redundant.